internal-packages/run-engine/src/engine/systems/dequeueSystem.ts

```
@@ -0,0 +1,620 @@
+              orgId,
+              projectId: run.runtimeEnvironment.projectId,
+              error: {
+                type: "INTERNAL_ERROR",
+                code: "TASK_RUN_DEQUEUED_MAX_RETRIES",
+                message: `We tried to dequeue the run the maximum number of times but it wouldn't start executing`,
+              },
+              tx: prisma,
+            });
+            //we don't need this, but it makes it clear we're in a loop here
+            continue;
+          }
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)